<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemplirBD</class>
 <widget class="QDialog" name="RemplirBD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de Finances</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabSpacing">
    <number>1</number>
   </property>
   <widget class="QWidget" name="ajouterPret">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>365</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Ajouter un Prêt</string>
    </attribute>
    <widget class="Line" name="linePret">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckValiderPret">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_19">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>265</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QLabel" name="dureePret">
        <property name="text">
         <string>Durée</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="debutTextPret">
        <property name="text">
         <string>début</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateDebutPret"/>
      </item>
      <item>
       <widget class="QLabel" name="finTextPret">
        <property name="text">
         <string>fin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateFinPret"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_20">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>95</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QLabel" name="textIdPret">
        <property name="text">
         <string>Id Prêt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineIDPret"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_21">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>177</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="textBanque">
        <property name="text">
         <string>Banque</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineBan"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_22">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <item>
       <widget class="QLabel" name="textMontant">
        <property name="text">
         <string>Montant</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineMont"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_23">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>169</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_23">
      <item>
       <widget class="QLabel" name="textInteret">
        <property name="text">
         <string>Intérêt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineInt"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_24">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>330</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <widget class="QPushButton" name="VidPret">
        <property name="text">
         <string>Vider les cases</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="enregistrerPret">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Enregistrer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="RefreshPret">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>70</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ApercuPret">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>70</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aperçu</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>251</width>
       <height>73</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="causePret">
        <property name="text">
         <string>Cause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="TextCausePret"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="TextSaisieID">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Donner un Id !</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/C:/Users/Cruzer/Desktop/attention.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QTableView" name="tabPretsAjout">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>110</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ajouterSponsor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>365</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Ajouter un Sponsor</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_25">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>114</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_25">
      <item>
       <widget class="QLabel" name="TextIdSpons">
        <property name="text">
         <string>Id Sponsor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineIDSponsor"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_26">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>330</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_26">
      <item>
       <widget class="QPushButton" name="VidSponsor">
        <property name="text">
         <string>Vider les cases</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="enregistrerSponsor">
        <property name="text">
         <string>Enregistrer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="CheckValiderSponsor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_27">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>265</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_27">
      <item>
       <widget class="QLabel" name="dureeSponsor">
        <property name="text">
         <string>Durée</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="debutTextSponsor">
        <property name="text">
         <string>début</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateDebutSponsor"/>
      </item>
      <item>
       <widget class="QLabel" name="finTextSponsor">
        <property name="text">
         <string>fin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateFinSponsor"/>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="lineSponsor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_28">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_28">
      <item>
       <widget class="QLabel" name="nomSponsor">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineNomSponsor"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_29">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_29">
      <item>
       <widget class="QLabel" name="SommeSponsor">
        <property name="text">
         <string>Somme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineSommeSponsor"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="RefreshSponsor">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ApercuSponsor">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aperçu</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabSponsorsAjout">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>90</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <zorder>tabSponsorsAjout</zorder>
    <zorder>layoutWidget_25</zorder>
    <zorder>layoutWidget_26</zorder>
    <zorder>CheckValiderSponsor</zorder>
    <zorder>layoutWidget_27</zorder>
    <zorder>lineSponsor</zorder>
    <zorder>layoutWidget_28</zorder>
    <zorder>layoutWidget_29</zorder>
    <zorder>RefreshSponsor</zorder>
    <zorder>ApercuSponsor</zorder>
   </widget>
   <widget class="QWidget" name="ajouterActionnaire">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>365</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Ajouter un Actionnaire</string>
    </attribute>
    <widget class="QCheckBox" name="CheckValiderActionnaire">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_30">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>129</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_30">
      <item>
       <widget class="QLabel" name="TextIdActionnaire">
        <property name="text">
         <string>Id Actionnaire</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineIDActionnaire"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_31">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_31">
      <item>
       <widget class="QLabel" name="nomActionnaire">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineNomActionnaire"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_32">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_32">
      <item>
       <widget class="QLabel" name="SommeActionnaire">
        <property name="text">
         <string>Somme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineSomActionnaire"/>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="lineActionnaire">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_33">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>330</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_33">
      <item>
       <widget class="QPushButton" name="VidActionnaire">
        <property name="text">
         <string>Vider les cases</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="enregistrerActionnaire">
        <property name="text">
         <string>Enregistrer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_34">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>134</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_34">
      <item>
       <widget class="QLabel" name="DivActionnaire">
        <property name="text">
         <string>Taux dividende</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineDivActionnaire"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="ApercuActionnaire">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aperçu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RefreshActionnaire">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableView" name="tabActionnairesAjout">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>220</y>
       <width>143</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="debutTextActionnaire">
        <property name="text">
         <string>Date Achat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateAchatActionnaire"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections/>
</ui>
